app/s3_client/s3_letter_upload_client.py

```diff
@@ -7,21 +7,34 @@ def get_transient_letter_file_location(service_id, upload_id):
     return 'service-{}/{}.pdf'.format(service_id, upload_id)
 
 
-def upload_letter_to_s3(data, file_location, status):
+def upload_letter_to_s3(data, file_location, status, page_count, filename):
```

Comment:
This method might now benefit from [requiring named arguments](https://stackoverflow.com/questions/2965271/forced-naming-of-parameters-in-python/14298976#14298976) to prevent someone calling it with the arguments in the wrong order (cool little trick that @CrystalPea taught me).
```suggestion
def upload_letter_to_s3(data, *, file_location, status, page_count, filename):
```

Reply:
👍 Good idea, updated to require named args now